packages/fiori-app-sub-generator/src/fiori-app-generator/transforms.ts

```diff
@@ -316,10 +315,7 @@ function getBaseAppConfig(
             id: appId,
             title: project.title,
             description: project.description,
-            flpAppId: getFlpId(
-                appId,
-                floorplan === FloorplanFF.FF_SIMPLE ? defaultNavActionDisplay : defaultNavActionTile
-            ),
+            flpAction: floorplan === FloorplanFF.FF_SIMPLE ? defaultNavActionDisplay : defaultNavActionTile,
```

Comment:
Can remove this entirely as the difference is writer specific and it could be done in the writer along with the flp id generation?